docs/xml/metainfo-component.xml

```diff
@@ -1106,6 +1106,58 @@
 				</listitem>
 			</varlistentry>
 
+			<varlistentry id="tag-relations-internet">
+				<term>&lt;internet/&gt;</term>
+				<listitem>
+				<para>
+					Require, recommend or support connectivity to the internet. The value of this item one of the following:
+				</para>
+				<itemizedlist>
+					<listitem><para><code>always</code> - Needs internet connectivity to work. If used in a <literal>recommends</literal> element, then this indicates that the app can work without internet, but the experience will be significantly degraded.</para></listitem>
+					<listitem><para><code>offline-only</code> - Never uses the internet, even if it’s available.</para></listitem>
```

Comment:
What is the value in case an application *can* use the internet to enhance the experience, but otherwise does not need to and would also work perfectly fine offline? (e.g. a media player fetching cover art for songs from the web, but which would play them perfectly find just as well w/o internet connection)
Would that be maybe:
```xml
<supports>
  <internet>always</internet>
</supports>
```